nxlemon/api/maps.pxd

```diff
@@ -0,0 +1,76 @@
+cdef extern from "lemon/concepts/maps.h" namespace "lemon:concepts":
+    cdef cppclass ReadMap[K, T]:
+        ctypedef K Key
+        ctypedef T Value
```

Comment:
These `ctypedef`s generate an error which says `expected an identifier, found 'ctypedef'`. Is there anything wrong with this?

Related C++ code : [here](https://github.com/networkx/networkx-lemon/blob/master/src/lemon/concepts/maps.h#L43-L48)
Extended log:
```sh
cdef extern from "lemon/concepts/maps.h" namespace "lemon:concepts":
    cdef cppclass ReadMap[K, T]:
        ctypedef K Key
       ^
------------------------------------------------------------

nxlemon/api/maps.pxd:3:8: Expected an identifier, found 'ctypedef'

Error compiling Cython file:
------------------------------------------------------------
...
cdef extern from "lemon/concepts/maps.h" namespace "lemon:concepts":
    cdef cppclass ReadMap[K, T]:
        ctypedef K Key
                  ^
```